.github/workflows/pull_request_no_chain.yml

```
@@ -17,7 +17,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Java and Maven Setup
-      uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/maven@main
+      uses: kiegroup/kie-ci/.ci/actions/maven@main
       with:
```

As we did on `main` and `7.67.x`:
```suggestion
      with:
        allow-snapshots: true
```